core/server/api/v2/config.js

```diff
@@ -0,0 +1,24 @@
+const {isPlainObject} = require('lodash');
+const config = require('../../config');
+const labs = require('../../services/labs');
+const ghostVersion = require('../../lib/ghost-version');
+
+module.exports = {
+    docName: 'config',
+
+    read: {
+        permissions: false,
+        query() {
+            return {
+                version: ghostVersion.full,
+                environment: config.get('env'),
+                database: config.get('database').client,
+                mail: isPlainObject(config.get('mail')) ? config.get('mail').transport : '',
+
+                labs: labs.getAll(),
+                clientExtensions: config.get('clientExtensions') || {},
+                enableDeveloperExperiments: config.get('enableDeveloperExperiments') || false
```

Comment:
@kevinansfield Let me know if you need more attrs

Reply:
> labs: labs.getAll(),

The labs values here feels like an exception 🤔 All fields are from config, but labs is actually from settings table. The original issue has not explained why we want to expose labs here, but just added it for now and followed the spec 👍